src/applications/static-pages/events/helpers/index.js

```diff
@@ -351,27 +351,45 @@ export const deriveEventLocations = event => {
     return locations;
   }
 
-  if (event?.fieldLocationHumanreadable) {
-    locations.push(event?.fieldLocationHumanreadable);
-  }
+  if (event?.fieldFacilityLocation?.entity?.fieldAddress) {
+    const fieldFacilityEntity = event?.fieldFacilityLocation?.entity;
+    if (fieldFacilityEntity?.fieldAddress?.addressLine1) {
```

Comment:
A lot of references to `fieldFacilityEntity?.fieldAddress` here. Maybe we could destruct `addressLine1`, addressLine2`, etc. off that variable at the top to lower the cognitive load here?